packages/waku/src/router/fs-router.ts

```diff
@@ -1,5 +1,5 @@
 import { unstable_getPlatformObject } from '../server.js';
-import { createPages } from './create-pages.js';
+import { new_createPages as createPages } from './create-pages.js';
```

Comment:
@dai-shi did you want to wait before switching fs-router over? or are we ready?

Reply:
I was thinking of migrating step by step, but if you are super confident, we can do altogether.